jck/jck.mk

```diff
@@ -569,73 +569,73 @@ ifeq (21, $(JDK_VERSION))
-   COMPILER_LANG_EXPR_TESTS_GROUP10=$(Q)lang/EXPR/expr753;lang/EXPR/expr754;lang/EXPR/expr755;lang/EXPR/expr756;lang/EXPR/expr757;lang/EXPR/expr758;lang/EXPR/expr759;lang/EXPR/expr760;lang/EXPR/expr761;lang/EXPR/expr762;lang/EXPR/expr763;lang/EXPR/expr764;lang/EXPR/expr765;lang/EXPR/expr768;lang/EXPR/expr777;lang/EXPR/expr790;lang/EXPR/expr791;lang/EXPR/expr792;lang/EXPR/expr793;lang/EXPR/expr794;lang/EXPR/expr795;lang/EXPR/expr796;lang/EXPR/expr797;lang/EXPR/expr798;lang/EXPR/expr799;lang/EXPR/expr800;lang/EXPR/expr801;lang/EXPR/expr802;lang/EXPR/expr803;lang/EXPR/expr804;lang/EXPR/expr805;lang/EXPR/expr806;lang/EXPR/expr807;lang/EXPR/expr809;lang/EXPR/expr810;lang/EXPR/expr811;lang/EXPR/expr812;lang/EXPR/expr813;lang/EXPR/expr814;lang/EXPR/expr815;lang/EXPR/expr816;lang/EXPR/expr817;lang/EXPR/expr818;lang/EXPR/expr819;lang/EXPR/expr820;lang/EXPR/expr821;lang/EXPR/expr822;lang/EXPR/expr823;lang/EXPR/expr824;lang/EXPR/expr825;lang/EXPR/expr826;lang/EXPR/expr827;lang/EXPR/expr828;lang/EXPR/expr829;lang/EXPR/expr830;lang/EXPR/expr831;lang/EXPR/expr832;lang/EXPR/expr833;lang/EXPR/expr834;lang/EXPR/expr835;lang/EXPR/expr836;lang/EXPR/expr837;lang/EXPR/expr839;lang/EXPR/expr840;lang/EXPR/expr841;lang/EXPR/expr842;lang/EXPR/expr843;lang/EXPR/expr844;lang/EXPR/expr845;lang/EXPR/expr846;lang/EXPR/expr847;lang/EXPR/expr848;lang/EXPR/expr849;lang/EXPR/expr850$(Q)
+   COMPILER_LANG_CLASS_TESTS_GROUP1=$(Q)lang/CLSS/clss001;lang/CLSS/clss004;lang/CLSS/clss006;lang/CLSS/clss008;lang/CLSS/clss009;lang/CLSS/clss010;lang/CLSS/clss012;lang/CLSS/clss015;lang/CLSS/clss016;lang/CLSS/clss018;lang/CLSS/clss019;lang/CLSS/clss020;lang/CLSS/clss021;lang/CLSS/clss022;lang/CLSS/clss023;lang/CLSS/clss024;lang/CLSS/clss025;lang/CLSS/clss026;lang/CLSS/clss027;lang/CLSS/clss028;lang/CLSS/clss029;lang/CLSS/clss030;lang/CLSS/clss031;lang/CLSS/clss033;lang/CLSS/clss034;lang/CLSS/clss035;lang/CLSS/clss038;lang/CLSS/clss044;lang/CLSS/clss045$(Q)
```

Comment:
`lang/CLSS/clss002;lang/CLSS/clss003;` are we going to remove them?

Reply:
re run splitter.sh against compiler-21 folder and updated the file.